notebook/tests/selenium/utils.py

```diff
@@ -100,6 +100,20 @@ def focus_cell(self, index=0):
         self.to_command_mode()
         self.current_cell = cell
 
+    def find_and_replace(self, index=0, find_txt='', replace_txt='', replace_all=False):
+        self.focus_cell(index)
+        esc(self.browser, 'F')
+        time.sleep(1)    # TODO: find better way to fill the find and replace form
+        self.browser.find_elements_by_css_selector("button.btn.btn-default.btn-sm")[2].click()
+        JS = "document.getElementsByClassName('form-control input-sm')[0].setAttribute('value', '%s')"%find_txt
```

Comment:
We shouldn't need to construct JS just to set the value of a form field like this, I think.

Reply:
right, let me see how I can make it better.